post-processor/manifest/post-processor.go

```diff
@@ -101,6 +111,9 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packer.Ui, source pa
 	artifact.BuilderType = p.config.PackerBuilderType
 	artifact.BuildName = p.config.PackerBuildName
 	artifact.BuildTime = time.Now().Unix()
+	if p.config.StripTime {
+		artifact.BuildTime = 0
+	}
```

Comment:
This allows to make my test based on the content of the `manifest.json` file. Without this `build_time` changes all the time.